integration/common/cmd.py

```diff
@@ -3,8 +3,10 @@
 import subprocess
 from os import path
 
-from common.constants import RETRY_COUNTS, RETRY_INTERVAL
-
+from common.constants import (  # NOQA
```

Comment:
Why adding `# NOQA` here?

Reply:
It's not necessary here. Will remove it.

Reply:
Updated. Thank you.